wdtk-client/src/main/java/org/wikidata/wdtk/client/Client.java

```diff
@@ -204,6 +214,32 @@ private void openActions() {
 	}
 
 	/**
+	 * Creates a report file including the results which came from the
+	 * {@link DumpProcessingAction#getReport()} methods or printing it to
+	 * stdout.
+	 * 
+	 * @throws IOException
+	 */
+	void createReport() throws IOException {
```

Comment:
Should be called "writeReport". Functions that start with "create..." should return a (newly created) object.